README.md

```diff
@@ -149,3 +150,81 @@ PostgreSQL configuration file after the benchmark completes.
 Some options can be passed on the command line and in a config file.
 Any such option passed on the command line will override the value set in the
 config file.
+
+
+# HTAP benchmark
+
+A mixed workload benchmark using a hybrid TPC-C/TPC-H schema is available in `benchmarks/htap`.
+Data preparation is identical to the other benchmarks (see "Creating a database and loading data"
+above).
+
+The HTAP benchmark requires command line arguments that differ from the ones described above.
+The `--dsn` argument is shared with the other benchmarks and must be provided.
+The `--benchmark` argument is not used, instead the name `htap` must be provided directly after the `--dsn` argument.
```

Comment:
Wouldn't it be more straightforward to have `htap` as a possible value for the `--benchmark` argument, like in data preparation?

Reply:
Yes, it would. But to keep the additional parameters separate we need a subparser, which can only be introduced with a positional argument like this. We could use a two-step parsing process, but that would complicate things more.

Reply:
I tried the 2-step parsing. It does work, but the problem is that if you use `--help`, the top-level parser does not collect the help texts from the lower level parsers. So you will see only help for the top-level parser's arguments, which would be `--benchmark`, and `--dsn` in this case.

Reply:
An alternative approach in order to enable the more consistent syntax would be to parse all arguments in one step and one parser. This of course with the downside of not keeping them separate in the codebase. It could still make sense, as the command line options can be seen as part of the UI.

Reply:
Yet another option: make all benchmarks use a positional argument instead of `--benchmark`.